app/src/main/java/org/wikipedia/games/onthisday/OnThisDayGameViewModel.kt

```diff
@@ -198,9 +198,10 @@ class OnThisDayGameViewModel(savedStateHandle: SavedStateHandle) : ViewModel() {
             "game_end"
         } else if (_gameState.value is Resource.Loading) {
             "game_loading"
-        } else {
+        } else if (::currentState.isInitialized.not()) {
+            "game_loading"
```

Comment:
After a discussion with Shay, please put "unknown" if the `_gameState` or `currentState` is `null`, for now.